extensions/libray-occupancy/package.json

```
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "libray-occupancy",
+  "title": "How Busy Is the Library Now",
```

```suggestion
  "title": "Library in University of St Andrews",
```
Can we come up with better naming of the extension?